assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -178,34 +166,15 @@ export default function SetupMainPAX( { finishSetup } ) {
 			<div className="googlesitekit-setup-module__step">
 				<AdBlockerWarning moduleSlug="ads" />
 
-				{ ! isAdBlockerActive &&
-					PAX_SETUP_STEP.FINISHED === showPaxAppStep && (
-						<div className="googlesitekit-setup-module__action">
-							<SpinnerButton
-								isSaving={ isNavigatingToOAuthURL }
-								disabled={
-									isNavigatingToOAuthURL ||
-									! paxAppRef?.current ||
-									! hasPaxSettings
-								}
-								onClick={ onCompleteSetup }
-							>
-								{ __( 'Complete setup', 'google-site-kit' ) }
-							</SpinnerButton>
-						</div>
-					) }
```

Comment:
As mentioned above, we no longer show our own "Complete setup" button and instead save things in the background, and then use the app's "Close" button to invoke the post-setup navigation.